main.go

```diff
@@ -185,6 +257,21 @@ func main() {
 			if installPage != "" {
 				publicInstallPages[filepath.Base(pth)] = installPage
 			}
+		case ".zip":
+			if xcarchive.CheckForXcarchive(pth) {
+				log.Donef("Uploading xcarchive file: %s", pth)
+
+				installPage, err := uploaders.DeployXcarchive(pth, config.BuildURL, config.APIToken, config.NotifyUserGroups, config.NotifyEmailList, config.IsPublicPageEnabled)
```

Comment:
xcarchive is not installable on device, so: `config.NotifyUserGroups, config.NotifyEmailList, config.IsPublicPageEnabled` parameters are shall not be used in this case, also the code block at the end

Reply:
Removed.